central/main.go

```diff
@@ -235,6 +236,22 @@ func main() {
 
 	devmode.StartOnDevBuilds("central")
 
+	if features.PostgresDatastore.Enabled() {
+		sourceMap, config, err := globaldb.GetPostgresConfig()
+		if err != nil {
+			log.Errorf("Unable to get Postgres DB config: %v", err)
+			return
+		}
+		// Check to see if a restore DB exists, if so use it.
+		if restore.CheckIfRestoreDBExists(config) {
+			// Now flip the restore DB to be the primary DB
+			err := restore.SwitchToRestoredDB(sourceMap, config)
```

Comment:
The restored db needs to go through migration process. If the restored version is earlier than the current version, we need to upgrade it.